prometheus/README.md

```diff
@@ -0,0 +1,22 @@
+# Indico Prometheus Plugin
+
+This plugin exposes a `/metrics` endpoint which provides Prometheus-compatible output.
+
+![](https://raw.githubusercontent.com/indico/indico-plugins/master/prometheus/screenshot.png)
+
+## prometheus.yml
+```yaml
+scrape_configs:
+  - job_name: indico_stats
+    metrics_path: /metrics
+    scheme: https
+    static_configs:
+      - targets:
+        - yourindicoserver.example.com
+    # it is recommended that you set a bearer token in the config
+    authorization:
+        credentials: inds_metrics_xxxxxx
+    # this is only needed in development setups
+    tls_config:
+      insecure_skip_verify: true
```

Comment:
Maybe better to default to false and mention that in development it may be necessary to set it to true? People tend to copy&paste and not change things that don't absolutely need to be changed :D